persist/sqlite/migrations.go

```diff
@@ -7,6 +7,15 @@ import (
 	"go.uber.org/zap"
 )
 
+func migrateVersion8(tx *txn, _ *zap.Logger) error {
+	_, err := tx.Exec(`CREATE TABLE signing_keys (
+	public_key BLOB PRIMARY KEY,
+	private_key BLOB UNIQUE NOT NULL
+);
+ALTER TABLE global_settings ADD COLUMN key_salt BLOB;`)
```

Comment:
We could consider `key_salt BLOB NOT NULL DEFAULT (RANDOMBLOB(32))`. It would (slightly) complicate the migration but a `NOT NULL` salt avoids a couple of awkward spots in the manager.

Reply:
I prefer to keep as much application logic as possible in the application over offloading it to ensure it's consistent between stores.